.github/workflows/compile-mermaid.yml

```diff
@@ -42,6 +42,8 @@ jobs:
     - name: Convert all svg files to png before uploading for automatic inspection
       run: |
         npm install convert-svg-to-png
+        # convert-svg-to-png tries to create files by changing their extension, which could clash with already-existing PNG files
+        for png_file in "${{env.INPUT_DATA}}"/*.png; do echo "Moving $png_file..."; mv "$png_file" "$png_file.bak"; done
```

Comment:
Please, remove this line.

Reply:
Done; This means `graph-with-br.mmd.png` that was created by `run-tests.sh`will be overwritten by the one created in this step, is that all right?
(I couldn't find what "automatic inspection" the percy step is performing, so I'm not exactly sure which PNG files I am supposed to upload to it :-S )